core/modules/css/KvInterface.h

```diff
@@ -42,18 +42,28 @@ namespace css {
 
 class KvInterface {
 public:
+
+    /**
+     * Sets a max length for kvKey.
+     * If it is changed it must also be changed in the
+     * CssData SQL Schema as well.
+     */
+    const unsigned int kMaxKeyLength = 255;
```

Comment:
And this does not compile without `static` on gcc44. Jenkins can be used to test building at >1 platform at a time: https://ci.lsst.codes/job/stack-os-matrix/build?delay=0sec